src/styles/sidebar/components/search-input.scss

```diff
@@ -0,0 +1,46 @@
+.search-input__form {
+  display: flex;
+  flex-flow: row nowrap;
+
+  position: relative;
+  color: $gray-dark;
+}
+
+.search-input__icon {
+  order: 0;
+}
+
+.search-input__input {
+  @include outline-on-keyboard-focus;
+
+  flex-grow: 1;
+  order: 1;
+
+  color: $text-color;
+
+  // Disable default browser styling for the input.
+  border: none;
+  padding: 0px;
+  width: 100%;
+
+  // The search box expands when focused, via a change in the
+  // `max-width` property. In Safari, the <input> will not accept
+  // focus if `max-width` is set to 0px so we set it to
+  // a near-zero positive value instead.
+  // See GH #2654
+  max-width: 0.1px;
+
+  transition: max-width .3s ease-out, padding-left .3s ease-out;
+
+  &:disabled {
+    background: none;
+    color: $gray-light;
+  }
+
+  // Expand the search input when focused (triggered by clicking
+  // on the search icon) or when `is-expanded` is applied.
+  &:focus,&.is-expanded {
```

Comment:
My eye wants to see a space after the comma here. Do we have linting for SCSS? And, as important: do you agree?

Reply:
My 2c is that we should use automated code formatting for any file type where there is a good automated formatted available and we have a substantial amount of code in that language. [Prettier](https://prettier.io/docs/en/index.html) supports SCSS apparently, so we could use it for that as well. This means the answer to your question is "whatever Prettier does" 😛